rtengine/iptransform.cc

```diff
@@ -519,6 +519,21 @@ bool ImProcFunctions::transCoord (int W, int H, const std::vector<Coord2D> &src,
             pLCPMap->correctDistortion(x_d, y_d, w2, h2);
         }
 
+        x_d /= params->perspective.camera_scale;
+        y_d /= params->perspective.camera_scale;
+        if (params->perspective.camera_defish) {
+            double const focal_source = params->perspective.camera_focal_length * maxRadius * 0.0462;
+            double focal_dst = focal_source;
+            x_d /= focal_dst;
+            y_d /= focal_dst;
+
+            double const r = std::sqrt(x_d*x_d + y_d*y_d);
+            double const factor = focal_source * std::atan(r)/r;
```

Comment:
Only nitpicking, but please add spaces to binary operators independent of the precedence. So this should read `x_d * x_d + y_d * y_d` and `focal_source * std::atan(r) / r`.

Reply:
Fixed.